src/Components/App/App.js

```diff
@@ -25,8 +25,8 @@ const App = () => {
   
   return (
     <>
-      {!user.token && <Login />}
-      {user.token &&
+      {!user.email && <Login />}
```

Comment:
Good idea on using Email for user conditional checks here! Thanks for all your help with this Oauth stuff @rwilliams659